### ISSUE TITLE
PDF exporting Messed Up

### ISSUE THREAD

**Issue by McToomin** [stolksdorf/homebrewery#174]
**Browser Type/Version**: Chrome version 52.0.2743.116 m

**Operating System**: Windows 7 Enterprise

**Issue Description**: I was working with the Homebrewery when everything switched, including a lot of margins. I spent a while updating them, but when I went to print to PDF, the columns and margins are not in the correct places.

I'll link to the current Homebrewery share page, vs. the PDF I printed out. I'll also link copy the entirety of the brew behind that.

**Related Images** : http://homebrewery.naturalcrit.com/share/H1ltwnPd

https://drive.google.com/open?id=0BzhFGMh1zxGHRlIxVWUtVEFudUU

~~~

>##### Warning!
><span style="display:block;margin-bottom:-10px;">This class requires you to have a standard deck of 52 playing cards in order to play!

&nbsp;

<div style='margin-top:65px'></div>

# <span style="display:block;margin-bottom:-6px;">Tarocchi
A cloaked man sits in a seedy tavern, a billow of smoke pouring from the folds of his robe, his booted feet rested upon a chair across from him. A fair maiden enters the room. Immediately, he tosses his hood back, and kicks the chair out from his table. "Please, take a seat," he invites, gesturing to the seat before beginning to lightly shuffle a deck of cards. He draws one, and places it on the table. "It seems like this is the company you were looking for..." She nods, and immediately moves to sit with him.

A party of adventurers stand at the feet of a mighty slain dragon. It was a hard fought battle, and even harder won. They look to their leader, who also lay fallen, gasping ragged breaths. A woman in loose robes approaches him, kneeling quietly. She smiles. "Don't worry," she says. "I had this saved just for you." She pulls a playing card out of her pocket and presses it against the fallen warrior. Instantly, his vitality returns, and the group begins to gather the gold they have earned.

A slight, older man sits in an ornate chair, behind and to the side of a great king. The kingdom has fallen into dire straights, and there is talk of rebellion. The king turns to his advisor, and asks how he can stop this problem before it gets worse. The old man smiles, adjusts his glasses, and pulls out a deck of cards. He begins laying them on a table in an indecipherable pattern. Finally, he places a single card into the middle of the pattern. He hesitates for just a moment before drawing the card. He smiles. "Don't worry, your highness," he says with great aplomb. "Your enemies should be the ones who are fearful. I forsee great things from your rule before the next phase of the moon..."

Tarocchi are mysterious wielders of magic, not using simple wands or staves, but drawing power from decks of cards. The luck of the draw, and the irony of not having a card you need, is never lost on a tarocchi, who is able to see the long term goal, and make use of what hand of cards they are dealt, both from their power packs, and from life itself.

### Power Pack
A tarocchi's most powerful weapon is their power pack, a deck of cards infused with magical abilities that only they can wield. Sometimes these power packs are legendary artifacts, with intricate artwork on the back, and court card depicting powerful rulers and tyrants, signifying these decks as special.

Often, however, these power packs simply appear as ordinary playing cards, allowing a tarocchi to feign innocence of their magical abilities. Indeed, power packs can be used as an ordinary deck of cards, and even if any of the cards are lost or destroyed, the power pack will reform in the tarocchi's deck by the light of the next morning, as arcane magic binds the two together.

&nbsp;

<div style='margin-top:163px'></div>

### Living as a Tarocchi
Unfortunatley for tarocchi, their magical abilities do not often lend themselves to finding steady work. The unreliability of their powers make others reluctant to employ them, or to rely on them. For this reason, tarocchi often regard their magical abilities as a sort of hobby, one which they must practice and hone in their free time, while finding work elsewhere.

The exception to this are tarocchi who are charismatic enough to find themselves in the employ of a thieve's guild, and who can convince the criminals with which they would work that their skills are valuable and reliable, so long as everyone else can be flexible. But to those who can be, a tarocchi is a powerful ally, capable of drawing new hope from their deck, even when all seems lost.

### Creating a Tarocchi
As you create your tarocchi character, think about your personal deck of cards. Where did they come from? Were they a gift, given to you by a family member or trusted friend who knew the secrets? Did you happen upon them, tossed away by someone afraid of their power? Did you purchase them from a mysterious merchant, whom no one else remembers?

You must also decide where your magical powers came from, as it is the blending of magic and a power pack that makes a tarocchi. Do you have an ancestral bloodline of magic users, whose arcane abilities have been born in you? Have you bonded with an otherworldly being, granting you powerful magic abilities? Or did you find your deck of cards and determine to learn more, poring over and studying ancienct texts to determine the true power of the cards?

### Quick Build
You can make a tarocchi quickly by following these suggestions. First, make Charisma your highest ability score, followed by Constitution. Second, choose the charlatan backgrond. Third, choose the *light*, *prestidigitation*, *ray of frost*, and *shocking grasp* cantrips, along with the 1st-level spells *charm person* and *magic missile*.

<img src='http://i.imgur.com/4BmMtCR.png' style='position:absolute;bottom:890px;left:-45px;' />

<img src='http://i.imgur.com/GO0kOrw.png' style='position:absolute;bottom:690px;right:-45px;' />

<div class='pageNumber auto'></div>
<div class='footnote'>TAROCCHI</div>

\page

<div class='classTable wide'>
##### The Tarocchi
|Level|Proficiency<br>Bonus|Features|Hand<br>Size|Cantrips<br>Known|Spells<br>Known|Spell<br>Level|
|:-:|:-:|:-|:-:|:-:|:-:|:-:|
|1st|+2|Power Pack, Game of Choice|5|4|2|1st|
|2nd|+2|New Deal, Trump Card (one use)|5|4|3|1st|
|3rd|+2|Ace in the Hole (one card), Lucky Draw|5|4|4|2nd|
|4th|+2|Ability Score Improvement|5|5|5|2nd|
|5th|+3|—|5|5|6|3rd|
|6th|+3|Game of Choice feature|6|5|7|3rd|
|7th|+3|—|6|5|8|4th|
|8th|+3|Ability Score Improvement|6|5|9|4th|
|9th|+4|—|6|5|10|5th|
|10th|+4|Game of Choice feature, Trump Card (two uses)|6|6|11|5th|
|11th|+4|—|6|6|12|6th|
|12th|+4|Ability Score Improvement|6|6|12|6th|
|13th|+5|—|6|6|13|7th|
|14th|+5|Game of Choice feature|7|6|13|7th|
|15th|+5|—|7|6|14|8th|
|16th|+5|Ability Score Improvement|7|6|14|8th|
|17th|+6|Ace in the Hole (two cards)|7|6|15|9th|
|18th|+6|Trump Card (three uses)|7|6|15|9th|
|19th|+6|Ability Score Improvement|7|6|15|9th|
|<span style="display:block;margin-bottom:-9px;">20th|<span style="display:block;margin-bottom:-9px;">+6|<span style="display:block;margin-bottom:-9px;">Stack the Deck|<span style="display:block;margin-bottom:-9px;">7|<span style="display:block;margin-bottom:-9px;">6|<span style="display:block;margin-bottom:-9px;">15|<span style="display:block;margin-bottom:-9px;">9th|
</div>

## Class Features
As a tarocchi, you gain the following class features.

#### <span style="display:block;margin-top:-4px;">Hit Points
___
- **Hit Dice:** 1d8 per tarocchi level
- **Hit Points at 1st Level:** 8 + your Constitution modifier
- **Hit Points at Higher Levels:** 1d8 (or 5) + your Constitution modifier per tarocchi level after 1st

#### <span style="display:block;margin-top:-4px;">Proficiencies
___
- **Armor:** Light armor
- **Weapons:** Simple weapons
- **Tools:** Gaming set (playing cards)
___
- **Saving Throws:** Wisdom, Charisma
- **Skills:** Choose two from Deception, History, Insight, Intimidation, Investigation, Perception, Performance, Persuasion, and Sleight of Hand

#### Equipment
<span style="display:block;margin-bottom:-3px;">You start with the following equipment, in addition to the equipment granted by your background:
- <span style="display:block;margin-top:-1px;">*(a)* a light crossbow and 20 bolts, or *(b)* any simple weapon
- <span style="display:block;margin-bottom:-1px;">*(a)* a dungeoneer's pack or *(b)* an explorer's pack
- A gaming set (playing cards)

Alternately, you can purchase your own starting equipment. At level 1, a tarocchi begins with 5d4 x 10 gp.

&nbsp;

<div style='margin-top:20px'></div>

### Power Pack
Some event in your past, whether it be through blood or study or making a pact with an otherworldly being, has left the indelible mark of arcane magic upon you. Along with a magical deck of cards that you have come into possession of, you are able to create powerful magical effects, so long as you draw the right card.

Unlike other magic users, you use your cards to fund your arcane abilities directly, meaning that you cast magic spells in a unique way.

#### Hand Size
Each card in your deck represents one spell slot. However, tarocchi spell slots work differently than those of other magic users.

At 1st level, your hand size is 5. You can draw up to your maximum hand size (as shown in the Hand Size column of the Tarocchi table) as a bonus action on your turn. Your hand persists until you finish a long rest. To cast a tarocchi spell, you must **play** a card of the appropriate spell slot from your hand. Unlike other magic users, you must use the specified card for each level spell, you cannot use higher cards to cast lower-level spells. The exception is cantrips, any card in the deck can be cast as a cantrip. Once you cast a spell, the card is **discarded** until you finish a long rest.

<div class='pageNumber auto'></div>
<div class='footnote'>TAROCCHI</div>

\page

###### <span style="display:block;margin-bottom:-11px;">&nbsp;

Your hand size increases as you gain levels, as shown in the Hand Size column of the Tarocchi table. You can **discard** cards from your hand at any time without an action. Once a card is discarded, it is lost until you finish a long rest, at which point all cards return to your deck. This includes cards which may have been left in other areas or even destroyed.

To cast a spell at a higher level, you must **play** additional cards when you cast the spell. For each card you play, the spell is cast one level higher, to a maximum of your highest spell level known as shown on the Spell Level column of the Tarocchi table.

For example, if you know the 1st-level spell *cure wounds*, and your hand includes a 2, 5, 10, Jack, and a King, you can **play** the 10 to cast *cure wounds*. Your other cards can be played as any cantrip that you know. If you wish to cast *cure wounds* as a 2nd level spell, you would **play** the 10 to cast the spell, along with any other card of your choice.

#### <span style="display:block;margin-top:-10px;">Cantrips
At 1st level, you know four cantrips of your choice from the tarocchi spell list. You learn additional tarocchi cantrips of your choice at higher levels, as shown in the Cantrips Known column of the Tarocchi table.

#### <span style="display:block;margin-top:-9px;">Spells Known of 1st Level and Higher
You know two 1st-level spells of your choice from the tarocchi spell list.

The Spells Known column of the Tarocchi table shows when you learn more tarocchi spells of your choice. Each of these spells can be no higher than the highest spell level you know, as shown in the Spell Level column of the Tarocchi table. For instance, when you reach 3rd level in this class, you can learn one new spell of 1st or 2nd level.

Additionally, when you gain a level in this class, you can choose one of the tarocchi spells you know and replace it with another spell from the tarocchi spell list, which also must be no higher than your higher spell level known.

#### <span style="display:block;margin-top:-10px;">Spellcasting Ability
Charisma is your spellcasting ability for your tarocchi spells, since the ability to use your magic comes from the supreme confidence and ability to improvise. You use your Charisma whenever a spell refers to your spellcasting ability. In addition, you use your Charisma modifier when setting the saving throw DC for a tarocchi spell you cast and when making an attack roll with one.

<center><span style="display:block;margin-bottom:5px;">**Spell save DC** = 8 + your proficiency bonus + your Charisma modifier</center>
<center><span style="display:block;margin-bottom:6px;">**Spell attack modifier** = your proficiency bonus + your Charisma modifier</center>

>##### <span style="display:block;margin-top:-6px;">Spellcasting Ability
><span style="display:block;margin-bottom:-10px;">The base class is written with a charismatic, charlatan type of character in mind. However, there are other kinds of card players as well such as the intellectual, calculating card counter. If it better fits your chosen flavor, you can use Intelligence as your spellcasting ability (as well as your saving throw proficiency) instead of Charisma.

#### Spellcasting Focus
The cards you play when you cast a spell count as a spellcasting focus for your tarocchi spells.

##### Card Slots
|Card|Suit|Spell Slot|
|:-|:-:|:-:|
|2 - 9|<span style="color:#ff0000">♦</span>, <span style="color:#ff0000">♥</span>, ♣, ♠|Cantrip|
|10|<span style="color:#ff0000">♦</span>, <span style="color:#ff0000">♥</span>, ♣, ♠|1st|
|Jack|<span style="color:#ff0000">♦</span>, <span style="color:#ff0000">♥</span>, ♣, ♠|2nd|
|Queen|<span style="color:#ff0000">♦</span>, <span style="color:#ff0000">♥</span>, ♣, ♠|3rd|
|King of Diamonds or Hearts|<span style="color:#ff0000">♦</span>, <span style="color:#ff0000">♥</span>|4th|
|King  of Clubs or Spades|♣, ♠|5th|
|Ace of Diamonds|<span style="color:#ff0000">♦</span>|6th|
|Ace of Hearts|<span style="color:#ff0000">♥</span>|7th|
|Ace of Clubs|♣|8th|
|Ace of Spades|♠|9th|

<div style='margin-top:21px'></div>

>##### <span style="display:block;margin-top:-6px;">Language of the Tarocchi
There are several terms that are unique to the tarocchi that are used in this supplement.
>
>***Rank.*** The **rank** of a card is its numerical value. So a card can have a rank of "3," which includes the 3 card in all four suits, or a rank of "9," or "king," and so on.
>
>***Discard.*** You can **discard** cards from your hand without playing them. These cards go to your discard pile, but do not activate any kind of magical effect.
>
><span style="display:block;margin-bottom:-14px;">***Play.*** When you **play** a card, you are activating its magical effect and (usually) casting a spell as you do so.

<div style='margin-top:23px'></div>

### Game of Choice
While a tarocchi is fond of any game involving cards, your must choose a game to focus your abilities on. Choose either Euchre, Mao, or Poker. Each choice affects your abilities and the way you are able to play your cards.

Your choice grants you features when you choose it at 1st level, and again at 6th, 10th, and 14th level.

### New Deal
Starting at 2nd level, any time you **play** a card from your hand, you can use your reaction to declare a New Deal. If you do so, take all of the remaining cards in your hand and shuffle them into your deck, and then draw a full hand.

You can use this feature a number of times equal to your Intelligence modifier (a minimum of once). You regain expended uses when you finish a short or long rest.

### Trump Card
Starting at 2nd level, you have the ability to play a card that may topple your foes before they have a chance to react. You can **play** a single card from your hand to cast a spell which has a casting time of 1 action or less and which targets a single creature as a reaction. You can only use this feature against a target which is surprised.

Once you use this feature, you must finish a short or long rest before you can use it again. Starting at 10th level, you can use it twice before a rest, and at 18th level, you can use it three times before a rest.

<div class='pageNumber auto'></div>
<div class='footnote'>TAROCCHI</div>

\page

### Ace in the Hole
Beginning at 3rd level, you have the ability to have a spell prepared even when your card is not ready. You can designate one card which remains in your Power Pack and cast a spell using that card (even though you do not have the card in hand). When you do draw the card from your Power Pack, **discard** it immediately and gain a new card, as its magical ability has been spent.

Once you use this feature, you must finish a long rest before you can use it again. Starting at 17th level, you can use it twice before a rest.

### <span style="display:block;margin-top:-6px;">Lucky Draw
Your magical abilities with cards give you some measure of luck over all decks of cards. Starting at 3rd level, whenever you draw a single card from a deck of cards, you can draw two cards and choose which card you want. The other card is shuffled back into the deck. This ability works with your Power Pack, but only when you draw a single card.

This is a magical ability. If a creature asks you to draw from a deck of cards, they do not see you draw two cards or make any kind of choice. You simply end up with the card you desire.

### <span style="display:block;margin-top:-6px;">Ability Score Improvement
When you reach 4th level, and again at 8th, 12th, 16th, and 19th level, you can increase one ability score of your choice by 2, or you can increase two ability scores of your choice by 1. As normal, you can't increase an ability score above 20 using this feature.

<div style='margin-top:2px'></div>

>##### <span style="display:block;margin-top:-6px;">Your Power Pack
>Like a wizard with a spellbook, your power pack is a physical object which can be lost or destroyed.
>
>***Replacing the Pack.*** Your power pack is typically a box, but it can also be some kind of special leather case or wrapping that keeps your cards together. So long as this housing is intact and complete, all of your cards will return to it when you finish a long rest. This includes situations where cards are lost or destroyed, they will still reform in your power pack.
>
>If the pack itself is destroyed, however, the magic of the cards ceases to exist. Your pack is a powerful magical item, and so is more resilient than it may appear. If the pack is destroyed, you will need to replace it. It takes 2 hours and 50 gp per spell level that you have access to. Once you have forged a new pack to hold your cards, they reform normally when you finish your next long rest.
>
>***The Pack's Appearance.*** Each power pack is unique, and are only typically recognized as special by other tarocchi. It may be a plain, functional set of leather straps which hold your cards together, or it may be an ornate magical box, inlaid with gold and intricate designs.
>
><span style="display:block;margin-bottom:-14px;">Your cards, like your power pack, are also highly variable. They may be very simple cards with plain backs and bare markings, or they may be incredibly detailed, with flourishes of artistic skill on the backs, and beautifully calligraphed symbols on the fronts.

&nbsp;

<div style='margin-top:48px'></div>

### Stack the Deck
Beginning at 20th level, you've learned to regain your most potent magics. Once per day when you finish a short rest, you can take any amount of a single **rank** card in your discard pile and shuffle them into your deck. This includes court cards, but not aces. The rank of "king" is split in half according to your spell slots. The king of diamonds (<span style="color:#ff0000"> ♦ </span>) and hearts (<span style="color:#ff0000"> ♥ </span>) count as a single rank, while the king of clubs ( ♣ ) and spades ( ♠ ) count as a separate rank.

## Tarocchi Spells
||||
|:-|:-|:-|
|<h5>Cantrips|<h5>2nd Level|<h5>4th Level|
|*chill touch*|*alter self*|*death ward*|
|*dancing lights*|*augury*|*dimension door*|
|*light*|*enhance ability*|*greater invisibility*|
|*mage hand*|*enlarge/reduce*|*ice storm*|
|*mending*|*gentle repose*|*locate creature*|
|*message*|*invisibility*|*polymorph*|
|*minor illusion*|*lesser restoration*|<h5>5th Level|
|*poison spray*|*locate animals or plants*|*contagion*|
|*prestidigitation*|*locate object*|*dream*|
|*produce flame*|*misty step*|*greater restoration*|
|*ray of frost*|*see invisibility*|*legend lore*|
|*resistance*|*scorching ray*|*modify memory*|
|*shocking grasp*|*silence*|*raise dead*|
|*spare the dying*|*spider climb*|<h5>6th Level|
|<h5>1st Level|*spiritual weapon*|*create undead*|
|*animal friendship*|*suggestion*|*find the path*|
|*charm person*|<h5>3rd Level|*globe of invulnerability*|
|*create or destroy water*|*animate dead*|*planar ally*|
|*cure wounds*|*bestow curse*|*true seeing*|
|*detect magic*|*clairvoyance*|<h5>7th Level|
|*disguise self*|*daylight*|*regenerate*|
|*heroism*|*dispel magic*|*resurrection*|
|*identify*|*fireball*|*symbol*|
|*illusory script*|*fly*|<h5>8th Level|
|*longstrider*|*glyph of warding*|*control weather*|
|*magic missile*|*lightning bolt*|*earthquake*|
|*shield*|*plant growth*|*glibness*|
|*silent image*|*remove curse*|<h5>9th Level|
|*sleep*|*revivify*|*foresight*|
|*speak with animals*|*sending*|*time stop*|
|*thunderwave*|*spirit guardians*|*wish*|

<img src='http://i.imgur.com/GO0kOrw.png' style='position:absolute;top:-1000px;right:-45px;' />

<div class='pageNumber auto'></div>
<div class='footnote'>TAROCCHI</div>

\page

<div style='margin-top:18px'></div>

## Game of Choice
Tarocchi love card games, but each tarocchi has a game they play more than others. This focus into their power pack gives them special abilities based on the type of game they play. Three games are presented below: Euchre, Poker, and Solitaire.

### Euchre
Euchre has gone by many different names, but the crux of the game is that it is a team game, with a group of facedown cards in the center of the board, referred to as the kitty.

Like your favorite game, you too work well with others as a euchre master, and learn to share cards with your friends to grant them powerful magical abilities.

#### Kitty Card
A euchre master's most powerful ability is that of the kitty card. A kitty card is one that you pour your magic into, but which can be saved and used at a later time.

Beginning at 1st level when you choose this game, you can **play** a card for a spell of 5th level or lower, naming the spell you cast into it. However, unlike a normal casting of the spell, you can identify the card as a kitty card. This traps the spell's magic in the card, but does not cast the spell immediately.

Once a card has been designated a kitty card, it can be given to any other creature to hold on to. At any time, that creature can touch the kitty card and say a code word (that you both come up with together). This allows the creature to cast the spell. The spell activates as though you were casting it. This includes using your level (if the card is a cantrip), or using your spell save DC for any targets which are affected by the spell. However, the range and area activate from the creature holding the card, which chooses any targets and makes any spell attack rolls required (although their roll still uses your ability modifier as normal).

You can have one kitty card active at a time. If you create a new kitty card before an existing one is activated, the old kitty card loses its power. Regardless of when it is created, all cards (including unexpended kitty cards) return to your deck when you finish a long rest.

#### Knock
Starting at 6th level, you have the ability to pass your turn for greater benefit later. You can choose to give up your action and all of your movement on your turn. If you do so, you gain one additional action on your next turn. You can only ever gain one additional action in this way.

#### Table Talk
Beginning at 10th level, you can communicate with your allies in secret. You can cast the *Rary's telepathic bond* spell, but only as a ritual.

#### Calling Trump
Starting at 14th level, when you use your Trump Card ability, instead of playing a card from your hand, you can instead cast the kitty card as a reaction. The kitty card activates from its current location, and you choose the targets and direction for any spells cast.

<span style="display:block;margin-bottom:-3px;">&nbsp;

### Mao
There is nothing more enjoyable to a Mao master than to introduce a new person to the game. Teaching someone to play is very simple:

><p align="right"><span style="display:block;margin-bottom:-10px;">*"The only rule I can tell you... is this one...*

Despite this seeming ambiguity, true Mao masters delight in following the secret rules of the game. Following these unspoken rules allows you great authority over your power pack.

#### Mao Begins Now
A Mao master constructs their discard pile differently than other tarocchi. While other tarocchi can simply discard their cards in any order, a Mao master must play their cards in a certain order. Once a Mao master **plays** their first card after a long rest, casting their first spell, new cards played must be played off of the previous cards. This is done in one of two ways: the next card played must either match the suit of the previous card played, or else it must match the rank of the previous card. If you **play** a card that does not match this convention on the previous card played, you gain one level of exhaustion. You can always **discard** cards without playing them.

So long as you play by the rules, however, your spells become more powerful. Each cantrip you play according to the rules which deals damage is cast with a damage increase, as though you were a higher level spellcaster. Each spell of 1st level and higher you play according to the rules is cast as a spell one level higher than normal (to your maximum spell level).

#### Reverse Order
Beginning at 6th level, you have the ability to reverse turn order temporarily. As a bonus action on your turn, you can invoke a Reverse command. Once your turn is complete, instead of the next creature in initiative taking its turn, the previous creature takes its turn. This continues, with initiative going in backward order until it returns to you. When it reaches your turn again, initiative continues as normal. Once you use this feature, you must complete a short or long rest before you can use it again.

#### Point of Order
Starting at 10th level, you can call upon an otherworldly presence to answer your questions. You can cast the *commune* spell, but only as a ritual. When you do so, a deity or extraplanary being that is friendly toward you appears to answer your questions.

#### Have a Nice Day
Just when an enemy thinks they've got you in their clutches, you can flip the rules of mao on its head. Beginning at 14th level, you can **play** any card in your hand without suffering the penalties of playing an incorrect card. Once you use this feature, you must complete a short or long rest before you can use it again.

\page

<div style='margin-top:63px'></div>

### Poker
Poker is a game with an extremely long history and thousands of variants. Most games of poker are played similarly, however. Each player has a hand, with which they try to have the best combination of cards. The player with the best hand wins the round, and play continues.

Your ability to form these hands not only wins games in the tavern, but also expounds your ability to cast spells through your power pack.

#### Winning Hand
As a poker player, you have the ability to play multiple cards at once to enhance your spell effects. While you can normally play additional cards to raise the spell level of the spells you cast, you can take it further by making poker hands, enhancing your spells even further.

At 1st level, you can play any of the following hands to gain an additional effect. You must **play** an entire poker hand of five cards, even if the Winning Hand you play does not require five cards. You must cast a single spell when you play the hand, which can be a spell from any of the cards in the hand. When you play five cards as a Winning Hand, it does not count as playing additional cards to raise the spell's level (although you may play additional cards *with* your Winning Hand to raise any spell levels as you desire).

##### Poker Pile
Unlike normal tarocchi, a poker player has two discard piles, one of which is referred to as the Poker Pile. When you play a Winning Hand, you use one (or more, if applicable) of the cards from the hand to cast the spell. This card is discarded normally. However, the remainder of the Winning Hand is placed into the separate Poker Pile. These are cards that were used to make a Winning Hand, but were not directly used to cast a spell.

When you finish a short or long rest, take all cards in the Poker Pile and shuffle them back into your Power Pack.

##### Winning Hands
The following combinations of cards can be played to form a Winning Hand.

***Two pair.*** This hand contains at least two cards of the same rank, and two cards of another rank. When you play this hand, you can cast two spells with a casting time of 1 action, one for each pair.

***Three of a kind.*** This hand contains at least three cards of the same rank. When you play this hand and cast a spell that has a casting time of 1 action, you can change the casting time to 1 bonus action for this casting.

***Straight.*** This hand contains at least five cards in sequential rank. When you play this hand, you can cast a spell one level higher than the highest spell card that was played (to a maximum of your highest spell level).

If your maximum spell level doesn't allow you to play a higher level spell, you can instead cast a spell you know as one level higher than normal (even if it's higher than your maximum spell level).

&nbsp;

<div style='margin-top:75px'></div>

&nbsp;&nbsp;&nbsp;&nbsp;***Flush.*** This hand contains at least five cards, all of which are the same suit. When you play this hand, all targets affected by the spell have disadvantage on the first saving throw made against the spell.

***Full house.*** This hand contains three cards of one rank, and two cards of another rank. When you play this hand, the spell you cast is increased by three levels (to a maximum of your highest spell level). You can then shuffle two of the cards from this hand back into your deck.

***Four of a kind.*** This hand contains four cards of the same rank. When you play this hand and cast a spell that targets only one creature and that doesn't have a range of self, you can target three other creatures in range with the spell.

***Straight flush.*** This hand contains five cards of sequential rank, all of which are the same suit. When you play this hand, you can cast a spell two levels higher than the highest spell card that was played (to a maximum of your highest spell level).

If your maximum spell level doesn't allow you to play a higher level spell, you can instead cast a spell you know as two levels higher than normal (even if it's higher than your maximum spell level).

#### Hold 'em
Beginning at 6th level, you can take your poker abilities a step further, invoking a variant called hold 'em.

On your turn, you can take a bonus action to **play** up to three cards in front of you as hold 'em cards. These cards remain in front of you for 1 minute, during which time you can draw back up to your normal hand limit. While they do not count against your hand limit, the hold 'em cards can be used in conjunction with cards in your hand to make a Winning Hand. They are considered played whenever you make the Winning Hand, and are discarded normally. If the hold 'em cards are not used within 1 minute, they are discarded.

#### Cozy Table
Starting at 10th level, you are ready for a round of poker no matter where you are. You can cast the *Leomund's tiny hut* spell, but only as a ritual. When you do so, you can also cast the *Tenser's floating disk* spell and the *unseen servant* spell as part of the same ritual. The disk and servant ar only active inside of the hut.

#### All In
Beginning at 14th level, you can choose one creature within 60 feet as your target. You and the creature each roll a d20. The higher of the rolls is the victor.

The victor of the roll has advantage on attack rolls against the loser. The loser has disadvantage on all saving throws from effects initiated by the victor.

This effect lasts for 1 minute. Once the effect ends, the target is immune to the effect for 24 hours, but you can initiate the effect against a new target.

<img src='http://i.imgur.com/4BmMtCR.png' style='position:absolute;bottom:930px;left:-45px;' />

<img src='http://i.imgur.com/GO0kOrw.png' style='position:absolute;bottom:900px;right:-45px;' />

<div class='pageNumber auto'></div>
<div class='footnote'>TAROCCHI</div>

\page

<div style='margin-top:528px'></div>

## Credits and Thanks
This supplement was produced, written by, and created by Alex Tanner. If you find it listed anywhere not authorized by the creator, please contact:

<a href="mailto:alexctanner@gmail.com">alexctanner@gmail.com</a>

### The Homebrewery
A huge thanks has to go out to [stolksdorf](https://www.reddit.com/user/stolksdorf) and [The Homebrewery](http://homebrewery.naturalcrit.com/) website, which made the creation of this piece possible in the first place! It's a fantastic resource, and there's nothing better for coming up with your own custom DnD content.

### Playtesters
All work I post goes through Reddit's [r/UnearthedArcana](https://www.reddit.com/r/UnearthedArcana/) community for balance and suggestions. I especially want to thank [Faen_Balinothe](https://www.reddit.com/user/Faen_Balinothe) and [DyscordianMalice](https://www.reddit.com/user/DyscordianMalice), who were able to perform several playtests which led to some very good changes for the class.

### Patreon
I do have a [Patreon page](https://www.patreon.com/alexctanner?ty=h) set up, where you can donate towards commissioning art. All profits made from my work goes toward commissioning art for the pieces. I very much appreciate all donations, and either way, thanks for reading!

<img src='http://i.imgur.com/4BmMtCR.png' style='position:absolute;bottom:490px;left:-45px;' />

<img src='http://i.imgur.com/GO0kOrw.png' style='position:absolute;top:-120px;right:-45px;' />

<div class='pageNumber auto'></div>
<div class='footnote'>TAROCCHI</div>

**Comment by McToomin** [comment]
I have since worked on the document in question, so I wanted to link to one I'm not currently working on as an example. I've got the Share page (which looks identical to my edit page), and the PDF it printed (with the messed up margins). I've asked around, and the issue as far as I can tell doesn't seem to be affecting others. I cleared my Chrome's cache with no effect.

http://homebrewery.naturalcrit.com/share/S1M5PmC5

https://drive.google.com/open?id=0BzhFGMh1zxGHbnZvRjFGMjhVRU0

~~~~~~~~~~~~~~~~~~~~~~~~~~~


### Creating a Dynamo

#### Quick Build
You can make a dynamo quickly by following these suggestions. First, put your highest ability score in either Strength or Dexterity, followed by Charisma. Second, choose the folk hero background. Third, choose the *heroism* and *inflict wounds* forces.



\page

<div class='classTable wide'>
##### The Dynamo<span style="float:right;margin:0;font-size:11px;">–Force Slots per Level–</span>
|Level|&nbsp;Proficiency&nbsp;<br>Bonus|&nbsp;&nbsp;Visceral&nbsp;&nbsp;<br>Power|Features|&nbsp;Supercharges&nbsp;|Forces<br>&nbsp;Known&nbsp;|&nbsp;1st&nbsp;|&nbsp;2nd&nbsp;|&nbsp;3rd&nbsp;|4th|
|:-:|:-:|:-:|:-|:-:|:-:|:-:|:-:|:-:|:-:|:-:|
|1st|+2|1d4|Supercharge, Unarmored Defense|1|2|1|–|–|–|
|2nd|+2|1d4|Helping Hand, Mild Mannered|1|3|1|–|–|–|
|3rd|+2|1d4|Hero Class|1|4|1|–|–|–|
|4th|+2|1d4|Ability Score Improvement|1|5|1|–|–|–|
|5th|+3|1d6|Extra Attack|2|6|1|1|–|–|
|6th|+3|1d6|Hero Class feature|2|7|1|1|–|–|
|7th|+3|1d6|Energy Burst, Uncontrollable Power|2|8|1|1|–|–|
|8th|+3|1d6|Ability Score Improvement|2|9|1|1|–|–|
|9th|+4|1d6|Power Blow (one use)|2|10|1|1|–|–|
|10th|+4|1d6|Hero Class feature|2|10|1|1|–|–|
|11th|+4|1d8|–|2|11|1|1|1|–|
|12th|+4|1d8|Ability Score Improvement|2|11|1|1|1|–|
|13th|+5|1d8|Unstoppable|2|12|1|1|1|–|
|14th|+5|1d8|Hero Class feature|2|12|1|1|1|–|
|15th|+5|1d8|Peak Form|3|13|1|1|1|–|
|16th|+5|1d8|Ability Score Improvement|3|13|1|1|1|–|
|17th|+6|1d10|–|3|14|1|1|1|1|
|18th|+6|1d10|Power Blow (two uses)|3|14|1|1|1|1|
|19th|+6|1d10|Ability Score Improvement|3|15|1|1|1|1|
|20th|+6|1d10|Legendary Hero|Unlimited|15|1|1|1|1|
</div>

## Class Features
As a dynamo, you gain the following class features.

#### Hit Points
___
- **Hit Dice:** 1d8 per dynamo level
- **Hit Points at 1st Level:** 8 + your Constitution modifier
- **Hit Points at Higher Levels:** 1d8 (or 5) + your Constitution modifier per dynamo level after 1st

#### Proficiencies
___
- **Armor:** Light armor, medium armor, shields
- **Weapons:** Four simple weapons and one martial weapon of your choice
- **Tools:** Choose one type of artisan's tools
___
- **Saving Throws:** Constitution, Charisma
- **Skills:** Choose two from Athletics, Acrobatics, Insight, Intimidation, Investigation, Persuasion, and Stealth

#### Equipment
You start with the following equipment, in addition to the equipment granted by your background:

- *(a)* leather armor or *(b)* chain mail
- *(a)* two simples weapons or *(b)* one martial weapon
- A set of artisan's tools of your choice
- An equipment pack of your choice

Alternately, you can purchase your own starting equipment. At level 1, a dynamo begins with 2d4 x 10 gp.

<div style='margin-top:57px'></div>

### Unarmored Defense
Starting at 1st level, while you are wearing no armor and not wielding a shield, your AC equals 10 + your Dexterity modifier + your Constitution modifier.

#### Visceral Power
You gain the following benefits while you are unarmed or wielding only weapons you gained proficiency with as a dynamo.

- You can use Dexterity instead of Strength for the attack and damage rolls of your unarmed strikes.
- You can roll a d4 in place of the normal damage of your unarmed strikes. This die changes as you gain dynamo levels, as shown in the Visceral Power table.
- When you use the Attack action with an unarmed strike on your turn, you can make one unarmed strike as a bonus action, assuming you haven't already taken a bonus action this turn.

<img src='http://i.imgur.com/4BmMtCR.png' style='position:absolute;bottom:500px;left:-45px;' />

<img src='http://i.imgur.com/GO0kOrw.png' style='position:absolute;bottom:500px;right:-45px;' />

<div class='pageNumber auto'></div>
<div class='footnote'>DYNAMO</div>

\page

<div style='margin-top:450px'></div>

### Supercharge
You may appear weak, but you have the ability to flood your body with power when provoked. On your turn, you can supercharge as an action. While supercharged, you gain the following benefits if you aren't wearing heavy armor.

- Your appearance changes as through the *disguise self* spell. You can change how your disguise appears when you finish a short or long rest, otherwise it remains the same each time you supercharge.
- The effects of all levels of exhaustion are suspended for the duration of your supercharge.
- You can take a bonus action to gain a number of temporary hit points equal to three times your dynamo level once per supercharge.
- You gain access to force slots, which allow you to channel powerful abilities according to your level. The Dynamo table shows how many force slots you have to channel forces into during your supercharge.

Your supercharge lasts for 1 minute. It ends early if you are knocked unconscious. You can also end your supercharge as a bonus action. You can initiate another supercharge as an action to extend the duration for 1 minute. You can change which forces you currently channel as part of this action.

You can supercharge a number of times as shown in the Supercharges column of the Dynamo table without penalty. When you finish a short or long rest, your number of available supercharges resets.

You can supercharge even if you have no uses remaining. Each time you supercharge beyond your number of available uses, you gain one level of exhaustion.

#### Force Slots
The Dynamo table shows how many force slots you have. The table also shows what the highest-level slot you have access to is; you have one slot of each level up to your highest-level slot known. To use a force power, you must channel it into a force slot during a supercharge. Each time you initiate a supercharge, you can choose which forces to channel into them.

For example, when you are 7th level, you have two force slots, one of 1st level, and one of 2nd level. When you initiate a supercharge, you can choose the 1st-level force *heroism*, and the 2nd-level force *invisibility*.

#### Forces Known of 1st Level and Higher
You know two 1st-level forces of your choice from the force list.

The Forces Known column of the Dynamo table shows when you learn more forces of your choice. Each of these forces must be of a level for which you have force slots. For instance, when you reach 5th level in this class, you can learn one new force of 1st or 2nd level.

Additionally, when you gain a level in this class, you can choose one of the forces you know and replace it with another force, which also must be of a level for which you have force slots.

#### Spellcasting Ability
Charisma is your ability modifier for any spells you know, so you use your Charisma whenever a spell refers to your spellcasting ability. In addition, you use your Charisma modifier when setting the saving throw DC for a spell and when making an attack roll with one.

<center><span style="display:block;margin-bottom:5px;">**Spell save DC** = 8 + your proficiency bonus + your Charisma modifier</center>
<center><span style="display:block;margin-bottom:9px;">**Spell attack modifier** = your proficiency bonus + your Charisma modifier</center>

#### Forces
Forces are spells that only affect you. When you channel a force into a force slot as part of your supercharge, the spell activates immediately on you. If a spell allows you to choose an effect (such as *enhance ability*), you must choose it when you initiate the supercharge. You do not need to concentrate on buffs to gain their benefit. The duration of a buff is equal to the length of your supercharge, at which point it immediately ends. If you take an action to extend your supercharge and do not channel the same buff (and effect, if applicable) into a force slot, that buff's benefits are lost immediately.

All forces can be channeled into a slot of a higher level. To channel a spell at a higher level, you must channel it into a force slot one level higher than normal for each level you raise the spell. Keep in mind that forces can only affect you, even if you raise the spell level.

### Helping Hand
Starting at 2nd level, you have the ability to protect and aid those around you. You can take a bonus action on each of your turns in combat to take the Help action.

<img src='http://i.imgur.com/4BmMtCR.png' style='position:absolute;bottom:550px;left:-45px;' />

<div class='pageNumber auto'></div>
<div class='footnote'>DYNAMO</div>

\page

<div style='margin-top:82px'></div>

### Mild Mannered
Although you are a great hero, you can attempt to hide your true identity. At 2nd level you gain proficiency in the Deception skill. Your proficiency bonus is doubled for any Deception check you make to attempt to hide your identity.

### Hero Class
At 3rd level, you choose a class of abilities that differentiate you from other dynamo. Choose Flying Brick, Lightning Bruiser, or Mighty Glacier. Your choice grants you features at 3rd level and again at 6th, 10th,  and 14th level.

### Ability Score Improvement
When you reach 4th level, and again at 8th, 12th, 16th, and 19th level, you can increase one ability score of your choice by 2, or you can increase two ability scores of your choice by 1. As normal, you can't increase an ability score above 20 using this feature.

### Extra Attack
Beginning at 5th level, you can attack twice, instead of once, whenever you take the attack action on your turn.

### Energy Burst
Starting at 7th level, you can initiate a supercharge as a reaction to you or an ally within 60 feet of you taking damage. Once you use this feature, you must finish a long rest before you can use it again.

### Uncontrollable Power
Beginning at 7th level, your unarmed strikes count as magical for the purpose of overcoming resistance and immunity to nonmagical attacks and damage.

### Power Blow
Starting at 9th level, you have the ability to hurl foes away from your allies. When an ally within 5 feet of you is damaged by a melee attack, you can take a reaction to make a weapon attack against the attacker. The target must succeed on a Strength saving throw with a DC equal to your force save DC. On a failure, you can push the target up to 15 feet away from you. You can use this feature once per supercharge, or twice per supercharge starting at 18th level.

### Unstoppable
Beginning at 13th level, you are not slowed down by difficult terrain. Additionally, you can move through a hostile creature's space, regardless of its size.

### Peak Form
At 15th level, your body becomes so powerful that you become immune to disease and poison.

&nbsp;

<div style='margin-top:370px'></div>

### Legendary Hero
Starting at 20th level, you have the ability to protect your allies from certain death. While supercharged, you can take a reaction whenever an ally within 60 feet of you takes damage to instead take that damage yourself.

Additionally, you cannot be reduced below 1 hit point during your supercharge, but the amount of additional damage you have taken accumulates and immediately hits you when your supercharge ends.

## Forces
Forces are sorted by their level.

>##### 1st-level Forces
>&nbsp;&nbsp;&nbsp;&nbsp;*armor of agathys*
>
>*comprehend languages*
>
>*heroism*
>
>*jump*
>
>##### 2nd-level Forces
>&nbsp;&nbsp;&nbsp;&nbsp;*alter self*
>
>*enhance ability*
>
>*invisibility*
>
>*spider climb*
>
>##### 3rd-level Forces
>&nbsp;&nbsp;&nbsp;&nbsp;*blink*
>
>*detect thoughts*
>
>*gaseous form*
>
>##### 4th-level Forces
>&nbsp;&nbsp;&nbsp;&nbsp;*fire shield*
>
><span style="display:block;margin-bottom:-10px;">*freedom of movement*

<img src='http://i.imgur.com/4BmMtCR.png' style='position:absolute;bottom:940px;left:-45px;' />

<img src='http://i.imgur.com/GO0kOrw.png' style='position:absolute;bottom:650px;right:-45px;' />

<div class='pageNumber auto'></div>
<div class='footnote'>DYNAMO</div>

\page

## Hero Classes
However one becomes a dynamo, certain patterns begin to emerge after a while. The names of these different kind of dynamo are colloquialisms of names given to ancient heroes of the past by the common folk, heroes whose true names have been lost to the passage of time.

No matter how you became a dynamo, your powers will tend toward certain patterns, placing you into a "class" of hero, the Flying Brick, Lightning Bruiser, or Mighty Glacier.

### Flying Brick


#### Expanded Force List
The powers of the Flying Brick let you choose from an expanded list of forces when you learn a new force. The following forces are added to the dynamo force list for you.

<div style='margin-top:5px'></div>

>##### 1st-level Forces
>&nbsp;&nbsp;&nbsp;&nbsp;*mage armor*
>
>##### 2nd-level Forces
>&nbsp;&nbsp;&nbsp;&nbsp;*see invisibility*
>
>##### 3rd-level Forces
>&nbsp;&nbsp;&nbsp;&nbsp;*water walk*
>
>##### 4th-level Forces
><span style="display:block;margin-bottom:-10px;">&nbsp;&nbsp;&nbsp;&nbsp;*fly*

<div style='margin-top:20px'></div>

#### Minor Forces
When you choose this archetype at 3rd level, choose two of the following cantrips. You can cast these cantrips at any time.

##### Flying Brick Minor Forces
|||
|:-|:-|:-|:-|
|*acid splash*|*fire bolt*|*mage hand*|*true strike*|

#### Winning Smile
Starting at 6th level, you can target each humanoid or beast that is aware of you within 120 feet during your supercharge. If the target can see you, it must succeed on a Wisdom saving throw (DC equal to your spell save DC) or be magically charmed. A charmed creature regards you as a trusted friend to be heeded and protected. Although the targe tisn't under your direct control, it takes your requests or actions in the most favorable way it can.

Each time you or your allies do anything harmful to the target, it can repeat the saving throw, ending the effect on itself on a success. Otherwise, the effect lasts the length of your supercharge, or until you end the effect as a bonus action. If a target's saving throw is successful, it is immune to your Winning Smile for the next 24 hours.

Once you use this feature, you must finish a short or long rest before you can use it again.

#### Great Strength
Beginning at 10th level, you gain proficiency in Strength saving throws.

#### True Strength
Starting at 14th level, once per supercharge you can take a bonus action to put power into your strike. If you do so, add doubled your Strength modifier in damage to the first successful attack you make before the end of your next turn. The target must succeed on a Strength saving throw (DC equal to your spell save DC). On a failure, the target is hurled a number of feet away from you equal to 10 times your Strength modifier (minimum 1). On a success, the target doesn't move.

### Lightning Bruiser


#### Expanded Force List
The powers of the Lightning Bruiser let you choose from an expanded list of forces when you learn a new force. The following forces are added to the dynamo force list for you.

<div style='margin-top:5px'></div>

>##### 1st-level Forces
>&nbsp;&nbsp;&nbsp;&nbsp;*longstrider*
>
>##### 2nd-level Forces
>&nbsp;&nbsp;&nbsp;&nbsp;*expeditious retreat*
>
>##### 3rd-level Forces
>&nbsp;&nbsp;&nbsp;&nbsp;*blur*
>
>##### 4th-level Forces
><span style="display:block;margin-bottom:-10px;">&nbsp;&nbsp;&nbsp;&nbsp;*haste*

<div style='margin-top:20px'></div>

#### Minor Forces
When you choose this archetype at 3rd level, choose two of the following cantrips. You can cast these cantrips at any time.

##### Lightning Bruiser Minor Forces
|||
|:-|:-|:-|:-|
|*dancing lights*|*light*|*ray of frost*|*shocking grasp*|

#### Flash
Starting at 6th level, you can take a bonus action on each of your turns when you are supercharged to take the Dash action.

#### Superb Reflexes
Beginning at 10th level, you gain proficiency in Dexterity saving throws.

#### Blinding Speed
Starting at 14th level, at the end of your turn during your supercharge you can choose to take another turn. You gain your full movement to use, and can make an additional action and bonus action. You instantly gain one level of exhaustion when you use this feature.

\page

### Mighty Glacier


#### Expanded Force List
The powers of the Mighty Glacier let you choose from an expanded list of forces when you learn a new force. The following forces are added to the dynamo force list for you.

<div style='margin-top:5px'></div>

>##### 1st-level Forces
>&nbsp;&nbsp;&nbsp;&nbsp;*false life*
>
>##### 2nd-level Forces
>&nbsp;&nbsp;&nbsp;&nbsp;*enlarge/reduce*
>
>##### 3rd-level Forces
>&nbsp;&nbsp;&nbsp;&nbsp;*protection from energy*
>
>##### 4th-level Forces
><span style="display:block;margin-bottom:-10px;">&nbsp;&nbsp;&nbsp;&nbsp;*stoneskin*

<div style='margin-top:20px'></div>

#### Minor Forces
When you choose this archetype at 3rd level, choose two of the following cantrips. You can cast these cantrips at any time.

##### Mighty Glacier Minor Forces
|||
|:-|:-|:-|:-|
|*chill touch*|*mending*|*poison spray*|*resistance*|

#### Barrel Through
Starting at 6th level, while you are supercharged you gain resistance to any damage from opportunity attacks. While supercharged you also have advantage on grapple checks, and your movement speed is not halved while you have a target grappled.

#### Great Fortitude
Beginning at 10th level, you add double your proficiency bonus to all Constitution saving throws.

#### True Might
Starting at 14th level, when you are supercharged and take damage, you can use your reaction to instead roll any number of your Hit Dice. Roll the dice and add your Constitution modifier to each roll as normal. Add together the result, and subtract that result from the amount of damage you would take.

**Comment by stolksdorf** [comment]
Your current brew has content that spills into a third column, 
![image](https://cloud.githubusercontent.com/assets/387742/18135732/e1d9e492-6f70-11e6-969c-3f54b6a4f6f6.png).

Chromes PDF exporter dones;t seem to like this and renders it as this. 
![image](https://cloud.githubusercontent.com/assets/387742/18135758/f590c906-6f70-11e6-9b90-8ee3465694dc.png)

Try modifying your brew to not have content spill over into the third column and see if this issue persists.

**Comment by McToomin** [comment]
I'll attach what that page looks like on my edit page as well as my share
page. On my edit page and share page, everything looks totally fine. It's
only when I go to print to PDF that I can see where the problem actually is.

**Comment by McToomin** [comment]
Also, I'm not sure if this helps, but I do remember I was literally in the
middle of working on a piece when the switch happened and all my columns
messed up. As it is, when I put my cursor all the way to the right of the
screen, it doesn't touch the scroll bar, I need to bring it in a bit. I was
hoping you have some idea of what is causing this, because I don't have a
clue, and this is the only website that's not acting normally.

**Comment by McToomin** [comment]
So I was able to fix the issue by uninstalling and reinstalling Chrome on my comp. No idea what could cause that, or why only the homebrewery seemed to be affected. But as of now, the issue seems to be fixed. Thanks!

**Comment by stolksdorf** [comment]
Thas really weird, but I'm glad it's working for you now? If you run into this again, let me know.
